PayItForward.Data.Services/Abstraction/IUsersService.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace PayItForward.Services.Data.Abstraction
+{
+    using System.Linq;
+    using System.Threading.Tasks;
+    using Dbmodel = PayItForward.Data.Models;
+
+    public interface IUsersService
+    {
+        // to be changed based on Controller needs
+        IQueryable<Dbmodel.User> GetAll();
```

Comment:
You should use another model for user, to not expose models from database outside of the the service layer